test/e2e/utils/typeormLoader.ts

```diff
@@ -0,0 +1,15 @@
+import { createDatabaseConnection } from './../../integration/utils/database';
+import { Connection } from 'typeorm/connection/Connection';
+import { createConnection } from 'typeorm';
+import { MicroframeworkSettings, MicroframeworkLoader } from 'microframework-w3tec';
+import { env } from '../../../src/core/env';
```

Comment:
I think it's unused. Please fix your tslint or ts => https://palantir.github.io/tslint/rules/no-unused-variable/